Sources/PackageModel/Target.swift

```diff
@@ -205,7 +213,7 @@ public class Target: PolymorphicCodableProtocol {
     }
 
     private enum CodingKeys: String, CodingKey {
-        case name, bundleName, defaultLocalization, platforms, type, sources, resources, ignored, others, buildSettings, pluginUsages
+        case name, potentialBundleName, defaultLocalization, platforms, type, path, sources, resources, ignored, others, buildSettings, pluginUsages
```

Comment:
Any compatibility issues or is this use of Codable just a cache?

Reply:
My understanding is that it is just a cache.